prototypes/entity.lua

```diff
@@ -107,13 +207,111 @@ data:extend({
 		{
 			shadow =
 			{
-				red = {0.26, -0.6},
-				green = {0.36, -0.6}
+				red = {0.56, -0.6},
+				green = {0.26, -0.6}
 			},
 			wire =
 			{
-				red = {-0.16, -0.9},
-				green = {0.16, -0.9}
+				red = {0.16, -0.9},
+				green = {-0.16, -0.9}
+			}
+		},
+	},
+	-- small storehouse
+	{
+		type = "container",
+		name = "storehouse-basic-small",
+		icon = "__Warehousing__/graphics/icons/storehouse-basic-small.png",
+		icon_size = 32,
+		flags = {"placeable-neutral", "placeable-player", "player-creation"},
+		minable = {mining_time = 2, result = "storehouse-basic-small"},
+		max_health = 250,
+		corpse = "big-remnants",
+		dying_explosion = "medium-explosion",
+		open_sound = { filename = "__base__/sound/machine-open.ogg", volume = 0.85 },
+		close_sound = { filename = "__base__/sound/machine-close.ogg", volume = 0.75 },
+		vehicle_impact_sound = { filename = "__base__/sound/car-metal-impact.ogg", volume = 0.65 },
+		resistances =
+		{
+			{
+				type = "fire",
+				percent = 90
+			}
+		},
+		collision_box = {{-0.95, -0.95}, {0.95, 0.95}},
+		selection_box = {{-0.95, -0.95}, {0.95, 0.95}},
+		fast_replaceable_group = "container",
+		inventory_size = storehouse_slots,
+		scale_info_icons = settings.startup["Warehousing-icon-scaling"].value,
+		picture =
+		{
+			filename = "__Warehousing__/graphics/entity/storehouse-basic-small.png",
+			priority = "high",
+			width = 80,
+			height = 62,
+			shift = {0.25, 0.0},
+		},
+		circuit_wire_max_distance = 7.5,
+		circuit_wire_connection_point =
+		{
+			shadow =
+			{
+				red = {.56, -0.3},
+				green = {.26, -0.3}
+			},
+			wire =
+			{
+				red = {0.16, -0.5},
+				green = {-0.16, -0.5}
+			}
+		},
+	},
+	-- tiny storehouse
+	{
+		type = "container",
+		name = "storehouse-basic-tiny",
+		icon = "__Warehousing__/graphics/icons/storehouse-basic-tiny.png",
+		icon_size = 32,
+		flags = {"placeable-neutral", "placeable-player", "player-creation"},
+		minable = {mining_time = 2, result = "storehouse-basic-tiny"},
+		max_health = 250,
+		corpse = "big-remnants",
+		dying_explosion = "medium-explosion",
+		open_sound = { filename = "__base__/sound/machine-open.ogg", volume = 0.85 },
+		close_sound = { filename = "__base__/sound/machine-close.ogg", volume = 0.75 },
+		vehicle_impact_sound = { filename = "__base__/sound/car-metal-impact.ogg", volume = 0.65 },
+		resistances =
+		{
+			{
+				type = "fire",
+				percent = 90
+			}
+		},
+		collision_box = {{-0.5, -0.5}, {0.5, 0.5}},
```

Comment:
The collision box is meant to be smaller than the selection box. See the `-basic` elements—their collision boxes are 0.3 units smaller than their selection boxes.

Reply:
Discussion will continue on another of these similar changes.